packages/vue/src/components/molecules/SfRange/SfRange.vue

```diff
@@ -0,0 +1,226 @@
+<template>
+  <div ref="range" class="sf-range" v-bind="$attrs" :disabled="disabled"></div>
+</template>
+<script>
+import noUiSlider from "nouislider";
+import wNumb from "wnumb";
+export default {
+  name: "SfRange",
+  props: {
+    /*
+     * Sets the number of sliders (1 or 2) by adding initial values to the array
+     */
+    slidersInitialValues: {
+      type: Array,
+      default: () => [0, 1],
+    },
+    /*
+     * Minimal value of the slider
+     */
+    min: {
+      type: Number,
+      default: 0,
+    },
+    /*
+     * Maximum value of the slider
+     */
+    max: {
+      type: Number,
+      default: 0,
+    },
+    /*
+     * The value of moving the slider on the scale
+     */
+    step: {
+      type: Number,
+      default: 1,
+    },
+    /*
+     * Disabling the slider
+     */
+    disabled: {
+      type: Boolean,
+      default: false,
+    },
+    /*
+     * Horizontal orientation if true, vertical orientation if false
+     */
+    horizontalOrientation: {
+      type: Boolean,
+      default: true,
+    },
+    /*
+     * Ltr orientation if true, rtl orientation if false
+     */
+    ltrDirection: {
+      type: Boolean,
+      default: true,
+    },
+    /*
+     * Boolean for showing the tooltips
+     */
+    tooltips: {
+      type: Boolean,
+      default: false,
+    },
+    /*
+     * Config for toolips formatting (from wNumb library):
+     * decimals - number of digits after dot/comma
+     * mark - sign separating the fraction value from integer
+     * thousand - sign separating the thousands from hundreds
+     * prefix - sign before value
+     * suffix - sign after the value
+     * negative - sign for negative values, by default it is set to "-"
+     */
+    formatTooltipsValues: {
+      type: Object,
+      default: () => {
+        return {
+          decimals: 2,
+          mark: ".",
+          thousand: " ",
+          prefix: "$",
+          suffix: "",
+          negative: "",
+        };
+      },
+    },
+  },
+  data() {
+    return {
+      config: {},
+    };
+  },
+  watch: {
+    min: {
+      handler(value) {
+        if (this.$refs.range) {
```

Comment:
You already have this condition inside `updateRangeOptions` method

Reply:
thanks, I changed this